apps/full-site-editing/bin/sync-blog-posts-block.sh

```diff
@@ -97,6 +97,10 @@ cp -R $CODE/src/blocks/homepage-articles $TARGET/blocks/
 cp -R $CODE/src/shared $TARGET/
 cp -R $CODE/src/components $TARGET/
 
+# Replace text domain
+find $TARGET/blocks/ -name \*.js -exec sed -i '' "s/, 'newspack-blocks' )/, 'full-site-editing' )/g" "{}" \;
+sed -i '' "s/'newspack-blocks',/'full-site-editing',/g" $TARGET/class-newspack-blocks.php
```

Comment:
`sed` ing is fine and fast — did you also consider running [`wp-textdomain`](https://www.npmjs.com/package/wp-textdomain) here? I'm assuming not worth for one file?

Would be great if it would handle .js files too, tho.

Reply:
should we remove the part of the syncing files on this process, in favor of https://github.com/Automattic/wp-calypso/pull/39196?

Reply:
I talked to @simison and he said his changes are independent of that `sed` above.

@simison, I agree but as of now `wp-textdomain` only supports PHP.